WoWsShipBuilder.Core/DataContainers/DispersionPlotHelper.cs

```diff
@@ -26,15 +26,15 @@ private static (double horizontalRadius, double verticalRadius) GetDispersionEll
         /// <summary>
         /// Calculates the projection ratio and the vertical radius length of the dispersion ellipse projected on water.
         /// </summary>
-        /// <param name="shell">Contains all the shell parameters.</param>
+        /// <param name="shell">The shell to calculate the <see cref="Ballistic"/> of.</param>
         /// <param name="maxRange">Max range a ship can fire at.</param>
         /// <param name="aimingRange">Range the ship is currently aiming at.</param>
         /// <param name="verticalRadius">The length of the dispersion ellipse vertical radius.</param>
         /// <returns>The projection ratio on water and on the perpendicular plane to the water, the length of the vertical radius projected on water and on the perpendicular plane to the water.</returns>
         private static (double waterLineProjection, double perpendicularToWaterProjection, double projectedOnWaterVerticalRadius, double perpendicularToWaterVerticalRadius) GetProjectedEllipse(ArtilleryShell shell, double maxRange, double aimingRange, double verticalRadius)
```

Comment:
Using tuples with that many parameters may justify using a dedicated type for it